src/magneticField/turbulentField/PlaneWaveTurbulence.cpp

```diff
@@ -169,6 +136,16 @@ PlaneWaveTurbulence::PlaneWaveTurbulence(const TurbulenceSpectrum &spectrum,
 
 		Vector3d kappa =
 		    Vector3d(sintheta * cos(phi), sintheta * sin(phi), costheta);
```

Comment:
I think it is ok to use a slightly different implementation compared to the paper, but we should comment on that shortly. I expect that people digging this deep into the code are able deal with that.

Reply:
So, I should just make that comment a bit shorter? (Or just remove the explanation for why this discrepancy exists?)

Reply:
I would probably just delete the last two sentences of that comment block. I like the reference to the GJ99 paper so people can go back and check the difference between the two.

Reply:
Thanks! Done.